src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -467,7 +467,11 @@ private function validateForm()
                 $fieldName = 'field' . $field['id'];
 
                 // skip
-                if ($field['type'] == 'submit' || $field['type'] == 'paragraph' || $field['type'] == 'heading') {
+                if ($field['type'] == 'submit' ||
```

Comment:
Eh, I think it's not. This is just personal preference if I can believe the internet (as in I havent found any real conclusion) PLUS this same style is used in the Backend Analytics Index :)